cves/CVE-2012-0022.yml

```diff
@@ -206,4 +215,10 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    This mistake is definitely no small oversight from indicidual code. It seems
```

Comment:
A search through the dictionary found no word: "indicidual", and I am unable to place which word was intended.